docs/API/canvas.mdx

```diff
@@ -30,8 +30,6 @@ const App = () => (
 | camera          | Props that go into the default camera, or your own `THREE.Camera`                                                                                 | `{ fov: 75, near: 0.1, far: 1000, position: [0, 0, 5] }` |
 | shadows         | Props that go into `gl.shadowMap`, can also be set true for `PCFsoft`                                                                             | `false`                                                  |
 | raycaster       | Props that go into the default raycaster                                                                                                          | `{}`                                                     |
-| vr              | Switches renderer to VR mode, then uses `gl.setAnimationLoop`                                                                                     | `false`                                                  |
-| mode            | React mode: legacy, blocking, concurrent                                                                                                          | `blocking`                                               |
```

Comment:
We'll want to explain the automatic WebXR and React rendering behavior in the v8 release notes.

Reply:
Maybe we should make a migration guide?

Reply:
Yes, we can drop a link to it. Should I add notes for the testing-renderer as well? Probably group it with the `render` API change.

Reply:
If you could that would be great, feel free to @ me for questions or requests and I can help where I can :)